scanner/Makefile

```diff
@@ -81,21 +100,21 @@ image-prefix := scanner-v4
 .PHONY: images
 images: scanner-image db-image
 
-scanner-go-build: deps
+image/scanner/bin/scanner: bin/scanner
```

Comment:
can we move this and `scanner-go-build-local` to the `Go Builds` section?

Reply:
Since the section is related to building images, I left them here. The scope of `scanner-go-build-local` is beyond the range of this PR and I left it untouched.